lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -3568,6 +3589,24 @@ namespace Js
         return (flags != ImplicitCall_None) || arrayIteratorPrototypeNext != scriptContext->GetLibrary()->GetArrayIteratorPrototypeBuiltinNextFunction();
     }
 
+    bool JavascriptLibrary::InitializeBigIntConstructor(DynamicObject* bigIntConstructor, DeferredTypeHandlerBase * typeHandler, DeferredInitializeMode mode)
+    {
+        typeHandler->Convert(bigIntConstructor, mode, 17);//what is 17???
+
+        // Old Note: Any new function addition/deletion/modification should also be updated in JavascriptLibrary::ProfilerRegisterNumber
```

Comment:
Seems like this is not relevant yet. The ProfilerRegister* methods (far below in this file) look like they are only dealing with methods, not plain data properties like what we have here. Eventually I imagine we'll add some methods and need a matching ProfilerRegisterBigInt function.

Reply:
If you plan to leave this comment (and the one in `InitializeBigIntPrototype`), could you please tag them with some easily searchable thing like `TODO (BigInt)` so we can easily find places that still need updates?

---
In reply to: [218132410](https://github.com/Microsoft/ChakraCore/pull/5706#discussion_r218132410) [](ancestors = 218132410)

Reply:
yes, make sense. Will do :)

---
In reply to: [218132911](https://github.com/Microsoft/ChakraCore/pull/5706#discussion_r218132911) [](ancestors = 218132911,218132410)